test/clj_kondo/namespace_config_test.clj

```diff
@@ -25,7 +26,9 @@
                      :type-mismatch {:level :error}}}
           "--config-dir" (str (fs/file "corpus" "namespace_config" ".clj-kondo"))
           "--cache" "true"))
-  (is (fs/exists? (fs/file "corpus" "namespace_config" ".clj-kondo" "inline-configs" "macros.clj" "config.edn")))
+  (let [inline-config (fs/file "corpus" "namespace_config" ".clj-kondo" "inline-configs" "macros.clj" "config.edn")]
+    (is (not (str/includes? (slurp inline-config) "#:" ))
```

Comment:
@lread Maybe leave the `fs/exists? ` check in as an explicit assert that the file should at least exist? It's not made implicit

Reply:
Sure, can do.